cli/ops/test_dispatcher.rs

```diff
@@ -0,0 +1,35 @@
+use crate::tools::test_runner::TestMessage;
+use deno_core::error::AnyError;
+use deno_core::serde_json;
```

Comment:
Nitpick: I think this module could be merged with `ops/test_runner.rs`

Reply:
> Nitpick: I think this module could be merged with ops/test_runner.rs

Agreed, merged into ops::testing (js equivalent is called testing).